configuration.go

```diff
@@ -132,6 +132,12 @@ type TagOn struct {
 	WorkflowDispatch WorkflowDispatchEmpty `yaml:"workflow_dispatch"`
 }
 
+type TestingOn struct {
+	PullRequest      Push                    `yaml:"pull_request"`
```

Comment:
Push and PR triggers have the same schema. I didn't want to rename the Push object and have all the importers of this go module have to deal with the rename on sdk-gen-config update